addon/usables/use-machine.ts

```diff
@@ -61,9 +68,9 @@ export class Statechart<
     // withConfig: Statechart<Context, Schema, Event>['withConfig'];
     // onTransition: Statechart<Context, Schema, Event>['onTransition'];
   } {
-    if (!this[INTERPRETER]) {
-      this._setupMachine();
-    }
+    // if (!this[INTERPRETER]) {
```

Comment:
why comment this out?

Reply:
I did the whole set up in `setup`, so there shouldn't be a case where we hit value and we haven't hit setup -- my understanding is that's baked into how the Resource API works.